copy_test.go

```diff
@@ -1007,6 +1007,35 @@ func TestStructToMap_NestedStruct_EmptyDst(t *testing.T) {
 	}, dst)
 }
 
+func TestStructToMap_NestedStruct_EmptyDst_OptionDst(t *testing.T) {
```

Comment:
Could you also add a similar test for `StructToStruct` please?

Reply:
I have not forgotten about this, I just haven't had the time :)

Reply:
Hey, I realized the same thing does not make any sense for struct to struct. I think I can add a similar feature, but that uses a certain proto field instead. Do you want me to implement the same `WithTag` but with a different behaviour for struct to struct? Different Options struct as well, maybe?

Reply:
Why do you think it won't make sense for StructToStruct?
I can imagine a situation where the destination struct has different from the source struct field names. 
Am i missing something...?

Reply:
Nevermind, I was thinking about it and just realized that I was being silly, and it was just because of the way I normally use this library that I didn't think it made sense! I just thought about it again. I'll implement this.